boards/holybro/durandal-v1/init/rc.board_defaults

```diff
@@ -5,3 +5,5 @@
 
 # Enable IMU thermal control
 param set-default SENS_EN_THERMAL 1
+
+param set UAVCAN_ENABLE 2
```

Comment:
This should already be set by default on stm32h7. https://github.com/PX4/PX4-Autopilot/blob/master/platforms/nuttx/init/stm32h7/rc.board_arch_defaults

Reply:
@dagar - I missed that :) 

fixed in last push